src/components/dock/dock-button/dock-button.scss

```diff
@@ -18,8 +18,11 @@
     padding: 0 0.5rem;
     min-width: var(--dock-item-height);
 
-    color: var(--button-text);
-    background-color: var(--button-background);
+    color: var(--limel-dock-item-text-color);
+    background-color: var(
+        --dock-item-background-color--deselected,
```

Comment:
I agree. Lucy and I also had this discussion. But since "unselected" didn't seem to be a "word" according to [Longman Dictionary](https://www.ldoceonline.com/spellcheck/english/?q=unselected) at least, we decided to go with [deselected](https://www.ldoceonline.com/dictionary/deselect).

Anyhow, maybe a better suffix would be `…--default` instead of deselected or unselected?

Reply:
Or, we don't need a suffix for the default value, and just have suffixes for the special cases, like `--selected` and `--recently-deselected`.

Reply:
OK, then let's remove it from here as well. We have never had anything for default or unselected. Can I quickly do it before you merge this?

Reply:
Ah, sorry, I missed this. Good thing it's still a private component, so we don't need to make it a breaking change! 😅